app/models/pame_evaluation.rb

```diff
@@ -336,6 +336,12 @@ def self.to_csv(json = nil)
 
     generate_csv(where_statement, restricted_where_statement)
   end
+
+  def self.last_csv_update_date
+    pame_data_csvs = Dir.glob("#{Rails.root}/lib/data/seeds/pame_data*")
+    latest_pame_csv = pame_data_csvs.sort.last.split('_').last
+    latest_pame_csv.split.first.to_date
```

Comment:
this is working well, but the `split.first` is having no effect on the string. Either we remove that bit (apparently it is smart enough to convert something like `"2022-03-01.csv"`) into a date anyway or we specify the splitter symbol (in this case `.`)

Reply:
added '.' pattern as split argument